From 13622ceccafce953f3e128021d2b026901e61bd9 Mon Sep 17 00:00:00 2001
From: Alex Naidis <alex.naidis@linux.com>
Date: Wed, 4 Jan 2017 22:33:44 +0100
Subject: [PATCH] SF: Enable SDCLANG LTO

Libsurfaceflinger consists of many source files
and is critical for performance.

Enabling SDCLANG LTO has a benefit here.

Change-Id: If52874e838985d76c923e611c47b748585b95dcf
Signed-off-by: Alex Naidis <alex.naidis@linux.com>
---
 services/surfaceflinger/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/services/surfaceflinger/Android.mk b/services/surfaceflinger/Android.mk
index 18cc946..65a5333 100644
--- a/services/surfaceflinger/Android.mk
+++ b/services/surfaceflinger/Android.mk
@@ -196,6 +196,8 @@ LOCAL_MODULE := libsurfaceflinger
 
 LOCAL_CFLAGS += -Wall -Werror -Wunused -Wunreachable-code
 
+LOCAL_SDCLANG_LTO := true
+
 include $(BUILD_SHARED_LIBRARY)
 
 ###############################################################
